asab/zookeeper/service.py

```diff
@@ -50,7 +50,7 @@ def DefaultContainer(self):
 
 
 	def build_container(self, config_section_name="asab:zookeeper"):
-		container = ZooKeeperContainer(self.App, config_section_name)
+		container = ZooKeeperContainer(self.App, config_section_name, z_path=None)
```

Comment:
`z_path=None` - this should not be necessary, it should be a default.

Reply:
I thought so too now. Thanks

Closing the PR.

Reply:
Closing this PR. Commits are redundent.